### DISCUSSION TITLE
How to Calculate Data

### DISCUSSION THREAD

**Discussion by flo-assling** [Budibase/budibase#10531] [Help] [answered]
I am currently using Budibase version 2.6.8 and I am getting frustrated after searching for hours how I can do simple calculations...

I added a MySQL datasource, and can create tables with the data I want.
One of my columns contains numbers, an article count.
How can I add a field which shows the sum of this column?

Thank you in advance!
Florian

**Comment by ADReece** [comment, marked as answer]
![image](https://github.com/Budibase/budibase/assets/103013045/e9f02f0f-2583-441b-8ad3-e83ac1fb4a80)

Are you familiar with the SUM MySQL function?
